lib/c_glib/src/thrift/c_glib/transport/thrift_platform_socket.h

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
I tottally agree. It's a reimplementation of the library for c_glib to not mix things. But this can be seen as an upgrade. And start making room for an common library after merge. Don't you think?